### ISSUE TITLE
View Configuration: Badges tab issues on mobile

### ISSUE THREAD

**Issue by SeanPM5** [home-assistant/frontend#4097]
**Checklist:**
- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Home Assistant release with the issue:**
0.100.2

**Browser and Operating System:**
iOS 13.1.3 / mobile Safari

**Description of problem:**
On mobile Safari after you add just one badge, it doesn't show you the "Entities" dropdown for the second badge. Well, it does, but it's rendered off-screen a bit and requires scrolling. And given that iOS hides the scrollbars,  and that more than half of the screen has nothing on it, the user would never think of needing to scroll...

Basically seems the Entities picker section is not changing height at all on mobile. At least in Safari. Although it *does* seem to change the height after you tap the Settings tab and then tap back to the Badges tab again. Just not if you remain on the Badges tab... Hopefully this pic explains it:

![badges](https://i.imgur.com/DxUG2Ly.jpg)

**Comment by SeanPM5** [comment]
Still an issue in 0.104.3

**Comment by SeanPM5** [comment]
Still an issue
